README.md

```diff
@@ -92,13 +107,11 @@ Are you interested in contributing to kops? We the maintainers and community wou
 
 ## Office Hours
 
-Kops maintainers also have time (1 hour) set aside every other week to offer help and guidance to the
-community. This is the maintainer's **office hours**. During this time we might work with newcomers, help with PRs, and discuss new features. Anything goes.
+Kops maintainers have 1 hour set aside every other week to discuss the current state of the kops project, to strategize about how to move forward, to discuss open and upcoming PRs, to demo, and to offer help and guidance to the community. This help tends to focus on developer needs, although we will never turn a courteous participant away. This hour is the maintainers' **office hours**.
```

Comment:
This hour is to help the developer community. Can we talk about the intention that we set on the call? Not sure how. Great already btw.